### ISSUE TITLE
why doesn't this replicate the paper's test accuracy?

### ISSUE THREAD

**Issue by jcpeterson** [jonnedtc/Shake-Shake-Keras#1]

**Comment by jonnedtc** [comment]
It's just a quick test and doesn't try to replicate the paper's test accuracy. Different network architecture, different optimizer, training for only 200 epochs instead of 1800 epochs, no hyper-parameter tuning, no learning rate annealing etc.
